docs/release-notes/8.4.0-1.md

```diff
@@ -1,2 +1,49 @@
 #  Percona XtraBackup 8.4.0-1 (2024-0-)
 
+Get started with [Quickstart Guide for Percona XtraBackup].
+
+## Release highlights
+
+Percona XtraBackup 8.4.0-1 is based on MySQL 8.4 Long Term Supported (LTS) Release. This release allows taking backups of MySQL 8.4 and MySQL 8.4-compatible servers.
```

Comment:
we need confirmation from QA to claim if PXB 8.4.0 is able to take backups (and prepare backups) of server versions 8.4.1, 8.4.2
@mohitj1988 ^ ?

Reply:
Hi @satya-bodapati - I confirm that PXB-8.4 is able to take backup of upstream 8.4.0, 84.1 and 8.4.2

Reply:
Thanks for verifying @mohitj1988!

Reply:
Thank you. Resolving this conversation